src/neuro_relay/server.py

```
@@ -251,20 +309,29 @@ async def _handle_intermediary_forward(self, msg: dict):
-                        print(True)
-                        return integration
-            return None
-        
-    async def send_to_connected_client(self, game_title: str, command_bytes: bytes):
+        for (
+            integration,
+            actions,
+        ) in self.intermediary.action_registry.items():
+            for action in actions:
+                if action_name in action.get("name", ""):
+                    print(True)
```

Ruff (0.14.8)

Boolean positional value in function call

(FBT003)